src/main/java/jmgomez/apipolicy/controller/PolicyController.java

```diff
@@ -1,40 +1,51 @@
 package jmgomez.apipolicy.controller;
 
 import jmgomez.apipolicy.model.Accident;
+import jmgomez.apipolicy.model.Policy;
+import jmgomez.apipolicy.model.dto.AccidentDto;
 import jmgomez.apipolicy.model.dto.PolicyDto;
-import jmgomez.apipolicy.model.dto.PolicyDtoCov;
 import jmgomez.apipolicy.service.PolicyService;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.security.core.context.SecurityContextHolder;
 import org.springframework.web.bind.annotation.*;
 
+import java.util.List;
+
 @RestController
 public class PolicyController {
 
     @Autowired
     private PolicyService policyService;
 
     @GetMapping("/policies")
-    public PolicyDto[] getPolicies() {
-        return policyService.getPolicies(getUserId());
+    public List<PolicyDto> getPolicies() {
+        return policyService.getPolicies(getUserId()).stream().map(this::changePolicyToPolicyDto).toList();
```

Comment:
changePolicyToPolicyDto is a bit weird. It's a public method but seems like private, innit? why calling it through this::? Naming could also be improved... map instead of change maybe? in fact, I'd like you to check the "mapping" concept. Usually there's something in spring for you in every matter. Lately we are using this: https://www.baeldung.com/mapstruct and you should easily see its power to keep a cleaner codebase